saylua/utils/email.py

```diff
@@ -14,7 +16,8 @@ def send_email(to, subject, text):
         data={"from": app.config.get('AUTOSEND_EMAIL'),
```

Comment:
Well, that too, but I was referring your LISP-ish dict format with the inlined brackets.